### DISCUSSION TITLE
Provide Build Scans for all projects

### DISCUSSION THREAD

**Discussion by hendrikebbers** [support-and-care/maven-support-and-care#39] [Ideas for Funding]
Apache provides https://ge.apache.org and all Maven projects should make use of it.

**Comment by rmannibucau** [comment]
What's the expected goal :thinking: ? Maybe a lack of perms but I didn't see much interesting data there for Maven projects we don't have somewhere already.

**Reply by hendrikebbers** [reply]
@ascheman has knowledge here. Looks like you get some nice insights.

**Reply by slachiewicz** [reply]
Thanks to people from Gradle we have PR with the proposal https://github.com/apache/maven/pull/1554

**Reply by olamy** [reply]
Agree. what values does this bring to us?
